docs/getting-started.md

```diff
@@ -58,17 +66,18 @@ to [the AKS installation guide](aks-installation.md).
 
 ## Importing Cloud VMs
 
-To manage security policies of Public Cloud VMs, we need to first import
-the target VMs onto `Nephe Controller`. The following section explains
-how to set up access to public cloud account, select target VMs, and
-import VMs into the K8s cluster as VirtualMachine CRs.
+To manage security policies of Public Cloud VMs, we need to first import the
+target VMs onto the `Nephe Controller`. The following section explains how to
+set up access to public cloud account, select target VMs, and import VMs into
+the K8s cluster as `VirtualMachine` CRs.
```

Comment:
Since not just CloudProviderAccount but also CloudEntitySelector need to be created in the Namespace, probably we should introduce the Namespace here, instead of in the CloudEntitySelector section. How about "and import VMs into a Namespace as`VirtualMachine` CRs. The sample commands import VMs into the `sample-ns` Namespace."?

Reply:
Done with slight modification.